help/include/manage-configured-topics.md

```diff
@@ -0,0 +1,30 @@
+{start_tabs}
+
+{tab|desktop-web}
+
+{settings_tab|topics}
+
+1. Configure notifications for each topic by selecting the desired option from
+   the dropdown in the **Status** column.
+
+{tab|mobile}
+
+1. Tap the **Streams**
+   (<img src="/static/images/help/mobile-hash-icon.svg" alt="hash" class="mobile-icon"/>)
+   tab at the bottom of the app.
+
+1. Select a stream containing topics you want to configure.
+
+1. Tap the **Topics list**
+   (<img src="/static/images/help/mobile-list-icon.svg" alt="list" class="mobile-icon"/>)
+   icon in the upper right corner of the app.
+
+{!topic-long-press-menu.md!}
+
+1. Tap **Mute topic** or **Unmute topic**.
+
+!!! tip ""
+
+    Muted topics are grayed out in the topics list.
```

Comment:
Same here, maybe keep the mobile tab instructions only in **Mute or unmute a topic**.